frontend/src/components/Uploader/ProvidersUploader.tsx

```diff
@@ -32,7 +32,7 @@ function onFileSelected(store: Store) {
       store.set('error')(e.toString())
     )
     store.set('uploadedProviders')(providers)
-    store.set('uploadedProvidersFilename')(file.name)
+    store.set('uploadedProvidersFilename')(file.name.slice(0, -4))
```

Comment:
Any reason not to keep the extension as part of the filename?

If we do want to remove it, can we do something more intelligent that would work for extensions of < or > 4 characters?

Reply:
For now it only accepts CSVs, but I added a todo